st2tests/tests/config.schema.yaml

```diff
@@ -0,0 +1,5 @@
+---
+config_item_one:
+  description: "Item use to test config context."
+  type: "string"
+  required: true
```

Comment:
What it work if you add a default value here?  Then you wouldn't need to add `st2tests/tests/configs/tests.yaml`.

Reply:
The tests pack I'm asking you to modify is located @ https://github.com/StackStorm/st2tests/tree/master/packs/tests.  It is a separate repo.  You're adding files in the wrong place.